mesheryctl/pkg/utils/healthcheck.go

```diff
@@ -0,0 +1,110 @@
+package utils
```

Comment:
LGTM @piyushsingariya 
// @navendu-pottekkat , i believe this should move to meshkit eventually, right?

Reply:
Yes, we can move this to meshkit in the future.